### ISSUE TITLE
How can I modify the code to apply knowledge distillation (not sequential KD)

### ISSUE THREAD

**Issue by haorannlp** [pytorch/fairseq#3443]
### Before asking:   
1. search the issues.   
2. search the docs.    

#### What is your question?
Hi! I try to apply knowledge distillation and I need to: 

1. load two training sets, i.e. original training set and the new training set distilled by the teacher model. Both data sets have the same amount of examples.
2. load the logits predicted by the teacher model. Because it can be time-consuming to let the teacher model produce the same logits again and again. 

Can you give me some advice on how to modify the source code to achieve that? Thank you!

#### What's your environment?

 - fairseq Version (e.g., 1.0 or master): 1.0
 - PyTorch Version (e.g., 1.0): 1.6
 - OS (e.g., Linux): Linux
 - How you installed fairseq (`pip`, source): source
 - Build command you used (if compiling from source):
 - Python version: 3.6
 - CUDA/cuDNN version:
 - GPU models and configuration:
 - Any other relevant information:
